seacrowd/sea_datasets/spamid_pair/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .utils.constants import Tasks
```

Comment:
I encountered error when running the test
```
    from .utils.constants import Tasks
ModuleNotFoundError: No module named 'seacrowd.sea_datasets.spamid_pair.utils'
```

I think we can leave this `__init__.py` file empty